lstmcpipe/scripts/script_merge_utils_copy_config.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+
+import os
+import shutil
+import argparse
+
+parser = argparse.ArgumentParser(description="Script to copy all the config files of the source directory into the "
+                                             "destination directory.")
+
+parser.add_argument('--source', '-s', type=str,
+                    dest='source',
+                    help='source argument of move_dir_content',
+                    )
+
+parser.add_argument('--destination', '-d', type=str,
+                    dest='dest',
+                    help='destination argument of move_dir_content',
+                    )
+
+
+def main():
+    args = parser.parse_args()
+
+    # HiPeRTA config is *.txt (<v4.0.0), otherwise is *.yml
+    config_files = [os.path.join(args.source, f) for f in os.listdir(args.source)
+                    if f.endswith(('.json', '.txt')) or f.startswith('hiperta_conf*.yml')]
```

Comment:
Does this expression make sense?
This selects json, txt and yml (only if it has hiperta in its name).
It specifically avoids other yml configs, why is that? The main config is yml as well.

Reply:
Hiperta config can be `.txt` or `.yml` (hiperta still supports both configs), and lstchain/ctapipe `.json`.

This expression looks for certain config files (the ones used in the r0_dl1 + merge stage and to be moved to `DL1` dir), to "maintain" a kind of provenance and copy them in the same dir as the merged DL1 files. (The merge stage is indeed an extension of r0_to_dl1, so basically to indicate the config used for this stage).

I think that copying the main (pipeline) configuration file in the DL1 dir is not really interesting, that's why it doesn't search for all the *.yml files.

Reply:
I think in the future it would be best to explicitly mention the files and cal the script in each stage instead of these conditions. Maybe some configs will be skipped if any of the projects decides to use another format.
But I guess its fine for now